java/native/src/CMakeLists.txt

```diff
@@ -1,34 +1,26 @@
-cmake_minimum_required(VERSION 2.8.0)
+################################################################
+# A minimal CMake file that is compatible with sbt-jni         #
+#                                                              #
+# All settings required by sbt-jni have been marked so, please #
+# add/modify/remove settings to build your specific library.   #
+################################################################
 
-set(ignoreMe "${SBT}") # sbt-jni defines -DSBT
-set(MAKE_COLOR_MAKEFILE ON)
+cmake_minimum_required(VERSION 2.8.0)
 
 # Define project and related variables
 # (required by sbt-jni) please use semantic versioning
 #
 project (gdaljni)
 set(PROJECT_VERSION_MAJOR 0)
 set(PROJECT_VERSION_MINOR 0)
-set(PROJECT_VERSION_PATCH 1)
-
-if (APPLE)
-  set(CMAKE_MACOSX_RPATH ON)
-  SET(CMAKE_SKIP_BUILD_RPATH TRUE)
-  SET(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
-  SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
-endif ()
+set(PROJECT_VERSION_PATCH 0)
 
 # Setup JNI
 find_package(JNI REQUIRED)
 if (JNI_FOUND)
     message (STATUS "JNI include directories: ${JNI_INCLUDE_DIRS}")
 endif()
 
-# Depend on GDAL
-find_package(GDAL REQUIRED)
```

Comment:
Ah it’s a gdal dep here; mb you want to leave the old cmakelists? It can be generated once and after that changes manually

Reply:
This comment is also valid for all C files in a native folder (: